mlflow/utils/requirements_utils.py

```diff
@@ -541,6 +541,16 @@ def _infer_requirements(model_uri, flavor, raise_on_error=False, extra_env_vars=
             unrecognized_packages,
         )
 
+    # Fix pandas incompatibility issue with numpy 2.x
```

Comment:
Can we include https://github.com/pandas-dev/pandas/issues/55519 in the comment?